coral-schema/src/test/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverterTests.java

```diff
@@ -612,6 +612,19 @@ public void testSelectStarFromNestComplex() {
         TestUtils.loadSchema("testSelectStarFromNestComplex-expected.avsc"));
   }
 
+  @Test
+  public void testSelectNestedStructFieldFromNestComplex() {
```

Comment:
can we add tests for deep nested fields (e.g., `struct.array_col`, `sturct.innter_struct.map_col`, ...)?